core/gpslocation.cpp

```diff
@@ -612,8 +612,8 @@ void GpsLocation::downloadFromServer()
 
 				struct gpsTracker gt;
 				gt.when = timestamp.toMSecsSinceEpoch() / 1000;
-				gt.latitude.udeg = latitude.toDouble() * 1000000;
-				gt.longitude.udeg = longitude.toDouble() * 1000000;
+				gt.latitude.udeg = lrint(latitude.toDouble() * 1000000);
+				gt.longitude.udeg = lrint(longitude.toDouble() * 1000000);
```

Comment:
I think the problem @neolit123 worries about is that the stored location could change, and how that might break assumptions in the current dive site management system.
Frankly, this is something where we should try to create a test case and figure out what happens if the rounding ends up turning into a ceil()...
But for now breaking this out of this commit seems like a better plan...